src/app/submit-item/components/submit/submit.component.html

```diff
@@ -134,12 +133,22 @@ <h4>Wie möchtest du dein Ergebnis erhalten?</h4>
 
 <ng-template #finish>
   <div class="page-container-column">
-    <h1>Danke, <span class="alt">dass du deinen Fall eingereicht hast!</span></h1>
+    <h1 *ngIf="!mailGiven">Danke, <span class="alt">dass du deinen Fall eingereicht hast!</span></h1>
+    <h1 *ngIf="mailGiven">Nur noch ein Schritt - <span class="alt"> Mailadresse bestätigen</span></h1>
     <div style="display: flex; flex-direction: row; width: 100%; height: 100%">
-      <div>
-        <p>
-          Unser Team von Detektiven prüft nun deinen Fall.<br/>
-          Sobald dein Fall gelöst wurde, erhältst du eine Benachrichtigung via E-Mail oder Handynummer.<br/><br/>
+      <div style="width: 60%; margin-right: 5%">
```

Comment:
@DavideMaiolo pointed out that inline style in html is a rather bad idea.
He proposed to create a class in the scss